dateparser/data/languages_info.py

```diff
@@ -206,6 +206,630 @@
     "tl"
 ]
 
+language_map = {
```

Comment:
I run the script again. Thanks
Done 👍🏻

Reply:
Hey @noviluni , I was unable to do that due to CLDR data issues even if I simply run `dateparser_scripts/order_languages.py` without `language_map` the following errors is thrown. So I copied `language_map` and pasted this data after running `dateparser_scripts/order_languages.py` and reversed all other changes

```
ERROR tests/test_data.py - ModuleNotFoundError: No module named 'dateparser.data.date_translation_data.jv'
ERROR tests/test_data.py
ERROR tests/test_language_detect.py - ModuleNotFoundError: No module named 'dateparser.data.date_translation_data.jv'
ERROR tests/test_language_detect.py
ERROR tests/test_search.py - ModuleNotFoundError: No module named 'dateparser.data.date_translation_data.jv'
ERROR tests/test_search.py
```

Note: this may be dues to old pinned CLDR data. As this error is not raised with the news CLDR data PR #941

Reply:
Hi @gavishpoddar, I'm not sure why are you having that issue, but it's working for me correctly when I try in your branch.

I added code suggestions with the changes. Could you directly apply them one by one through the graphical interface (button "commit suggestion) to avoid any issue?:
![imagen](https://user-images.githubusercontent.com/22377678/130326396-fb458494-a73b-4ff8-8810-c0df009e1baa.png)

Once you do it I will check it again and if nothing changes it means that it's ok :)